tests/testthat/test-survival-bayes.R

```diff
@@ -0,0 +1,301 @@
+
+test_that("survival analysis - tuning via Bayesian search", {
+  skip_on_cran()
+  skip_if_not_installed("modeldata")
+  skip_if_not_installed("censored")
+
+  suppressPackageStartupMessages(library(dplyr))
+  suppressPackageStartupMessages(library(censored))
+  suppressPackageStartupMessages(library(modeldata))
+  suppressPackageStartupMessages(library(rsample))
+  suppressPackageStartupMessages(library(yardstick))
+  suppressPackageStartupMessages(library(recipes))
+  suppressPackageStartupMessages(library(workflows))
+  suppressPackageStartupMessages(library(dials))
```

Comment:
I don't think those affect only the test so I think it's actually better to put them at the top of the file instead of inside a `test_that()` expression.